Doc/library/asyncio-eventloop.rst

```diff
@@ -701,6 +701,23 @@ Low-level socket operations
 
       :meth:`AbstractEventLoop.create_server` and :func:`start_server`.
 
+.. coroutinemethod:: AbstractEventLoop.sock_sendfile(sock, file, \
+                                                     offset=0, count=None, \
+                                                     *, fallback=True)
+
+   Send a file by using high-performance :mod:`os.sendfile` and return
+   the total number of bytes which were sent.
+
+   Modelled after blocking :meth:`socket.socket.sendfile`, see the
+   method for parameters meaning.
```

Comment:
"Asynchronous version of socket.sendfile." [ Just copy docs for "offset" and "count" parameters ]

Reply:
Also, document that "file" must be a file object, "sock" must be an instance of "socket.socket", and explain what "fallback" does.